file_paths.md

```diff
@@ -197,13 +210,25 @@ user or group.
             puppet.conf *                     # :config
             routes.yaml                       # :route_file
             ssl                               # :ssldir
-    -------------------------------------------------------------------------------------------------------               
+    -------------------------------------------------------------------------------------------------------
         var *
             log                               # :logdir                      same
                 puppet.log                    # not enabled by default
             run                               # :rundir                      same
                 agent.pid                     # :pidfile
 
+    C:\ProgramData\PuppetLabs\pxp-agent *
+        etc *
+            pxp-agent.conf                    # pxp-agent configuration file
```

Comment:
The question has come up before with respect to MCO and shipping a "default" config.  What is the approach with `pxp-agent`? See [MCO-727](https://tickets.puppetlabs.com/browse/MCO-727)